requirements/github-actions.txt

```diff
@@ -0,0 +1,8 @@
+#
+# This file is autogenerated by pip-compile with Python 3.12
+# by the following command:
+#
+#    pip-compile --allow-unsafe --config=../.pip-tools.toml --output-file=github-actions.txt --strip-extras github-actions.in
```

Comment:
Compiling PyYAML on various VMs might be problematic, so I'd pass `--only-binary=:all:` via pip-opts or even put it into the config. This way, installing would be more predictable on different versions of Ubuntu VMs that GitHub has.

Another concern is that originally, I opted for a docker-based action so that the external runtime is not mutated (that's one of a few reasons). By installing things with `pip` we violate that promise. I was kinda hoping that it'd be possible to avoid using a third-party dependency at all.

Reply:
If we go with the latest approach of dumping JSON to a `.yml` file, we won't need to deal with the PyYAML requirement.

Reply:
Marking this thread as resolved.